packages/python/plotly/plotly/io/_base_renderers.py

```diff
@@ -275,27 +273,13 @@ def activate(self):
                     )
                 )
 
-            if not self.requirejs:
-                raise ValueError("global_init is only supported with requirejs=True")
-
             if self.connected:
-                # Connected so we configure requirejs with the plotly CDN
                 script = """\
         <script type="text/javascript">
         {win_config}
         {mathjax_config}
-        if (typeof require !== 'undefined') {{
-        require.undef("plotly");
-        requirejs.config({{
-            paths: {{
-                'plotly': ['{plotly_cdn}']
-            }}
-        }});
-        require(['plotly'], function(Plotly) {{
-            window._Plotly = Plotly;
-        }});
-        }}
         </script>
+        <script src=\"{plotly_cdn}\" charset="utf-8"></script> 
```

Comment:
How about using a `script` tag with *module* type i.e. [native ES6 import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) to have async loading.

Something like: 
```html
<script type="module">import \"{plotly_cdn}\"</script>
```

Reply:
Sure